src/Util/IntrospectionProcessor.php

```diff
@@ -157,6 +158,10 @@ private function exceptionToContext(\Throwable $exception): array
             'line' => $exception->getLine(),
         ];
 
+        if ($exception instanceof AgentHttpException) {
+            $context['details'] = $exception->getDetails();
```

Comment:
can you add a test case?
```suggestion
            $context['details'] = $exception->getDetails()->jsonSerialize();
```